### ISSUE TITLE
Cream on pink in the subscribe error messaging doesn't meet contrast requirements

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#101]
## Description
Unfortunately <del>black</del>cream pink isn't a compliant color combination. 
<img width="850" alt="Screen Shot 2022-09-23 at 6 16 02 PM" src="https://user-images.githubusercontent.com/14154792/192063845-a0f0930d-2c16-434b-880d-ee495398e612.png">

## Suggestion
We could opt to use light pink with dark pink or dark maroon on yellow!

<img width="523" alt="Screen Shot 2022-09-23 at 6 17 27 PM" src="https://user-images.githubusercontent.com/14154792/192063976-cff65665-8b01-4c27-8861-edcae4e3c425.png">
<img width="306" alt="Screen Shot 2022-09-23 at 6 17 46 PM" src="https://user-images.githubusercontent.com/14154792/192064010-48faff2a-5022-49bd-98c5-0b1a83201509.png">
